cpp/benchmarks/io/cuio_common.cpp

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+#include "rmm/mr/pinned_host_memory_resource.hpp"
+#include "rmm/resource_ref.hpp"
```

Comment:
Use `<>`. clang-format will probably require you to move these, too.
```suggestion
#include <rmm/mr/pinned_host_memory_resource.hpp>
#include <rmm/resource_ref.hpp>
```